.github/workflows/build.yml

```diff
@@ -300,12 +300,19 @@ jobs:
           echo "PROJECT_NAME: $PROJECT_NAME"
           echo "project_name=$PROJECT_NAME" >> $GITHUB_OUTPUT
 
-      - name: Generate image full name
-        id: image-name
+      - name: Generate image name(s)
+        id: image-names
         env:
           IMAGE_TAG: ${{ steps.tag.outputs.image_tag }}
           PROJECT_NAME: ${{ steps.setup.outputs.project_name }}
-        run: echo "name=${_AZ_REGISTRY}/${PROJECT_NAME}:${IMAGE_TAG}" >> $GITHUB_OUTPUT
+          SHA: ${{ github.sha }}
+        run: |
+          NAMES="${_AZ_REGISTRY}/${PROJECT_NAME}:${IMAGE_TAG}"
+          # if [[ "${IMAGE_TAG}" == "dev" ]]; then
+          SHORT_SHA=$(echo "${SHA}" | cut -c1-7)
```

Comment:
Something like `git rev-parse --short` here would also work I think, and not be subject to possible changes in arguments via the `cut` command.